storage.go

```diff
@@ -325,6 +339,58 @@ func (s *Storage) keyList() (map[string]bool, uint64, error) {
 	}
 	return keyList, rev, nil
 }
+
+// keyListAdd adds the given key to the keylist
+func (s *Storage) keyListAdd(key string) error {
```

Comment:
are these operations thread safe?

Reply:
Covered in https://github.com/SkynetLabs/certmagic-storage-skydb/pull/3#discussion_r658682278